### ISSUE TITLE
No aut-select unselect delegate available

### ISSUE THREAD

**Issue by danjayne** [tochoromero/aurelia-table#43]
As far as I can see, there's no way to observe when a row has been deselected. I would like to show an edit form on select and hide the form when the same row is deselected.

**Comment by danjayne** [comment]
I have a very simple scenario:

```
let items = [
    { firstName: "Joe", surname: "Bloggs"}, 
    {firstName: "Jim", surname: "Bloggs"}
];

```
```
<table if.bind="items" class="table table-striped" aurelia-table="data.bind: items;> 
...
</table>
```

Click on an item in the table - say Joe Bloggs

I then want a form to appear to edit Joe Bloggs which I'm achieving by having a select.delegate on the table row repeater.

However, when I **unselect** the row, I would like to be able to hide the form which I can't currently do.

**Comment by abeam-intricity** [comment]
As a workaround, I'm using `click.delegate="rowSelected($event)"` on the tr. I can then do a quick loop to check how many items in the bound element have `$isSelected == true`.

**Comment by abowen** [comment]
Workaround doesn't work unfortunately when using `custom.bind`

Instead I've avoided using aut-select, and instead do
`<input type="checkbox" checked.bind="item.isSelected" change.delegate="selectionChanged(item)"/>`

Then in typescript
`item.isSelected = !item.isSelected;`

**Comment by wernerfletcher** [comment]
Agreed, not having a deselect event is very frustrating.  Using `select.delegate` on the `tr`, but it only fires when the row has not been selected yet.  When you click the selected `tr` to deselect, it does not fire the event again.
